apps/vor/pkg/school/school.go

```diff
@@ -83,7 +85,87 @@ func getClasses(server rest.Server, store postgres.SchoolStore) http.Handler {
 		return nil
 	})
 }
+func getClassAttendance(server rest.Server, store postgres.SchoolStore, classStore postgres.ClassStore) http.Handler {
```

Comment:
Yeah, i think this is getting a bit too complicated for now. I think it will be fine to only get the attendance data for one given date. 
So this would just returns something like `[]attendanceData`.

Currently, I don't think any ui needs all the attendance data at once.